beancount-tests.el

```diff
@@ -333,13 +334,15 @@ known option nmaes."
     (goto-char 7)
     (should (equal (thing-at-point 'beancount-link) "^link"))))
 
+;;; Date shifting
+
 (ert-deftest beancount/date-shift-up-day ()
-    :tags '(date-shift)
-    (with-temp-buffer
-      (insert "2024-05-11\n")
-      (goto-char 0)
-      (beancount-date-up-day)
-      (should (equal (thing-at-point 'line) "2024-05-12\n"))))
+  :tags '(date-shift)
+  (with-temp-buffer
+    (insert "2024-05-11\n")
+    (goto-char 0)
+    (beancount-date-up-day)
+    (should (equal (thing-at-point 'line) "2024-05-12\n"))))
```

Comment:
I think that this adjustment of the indentation should be applied to the test right below this as well.